### PR TITLE
Lower Default API_TIMEOUT to 10s

### PR DESCRIPTION
**Pull request by austinborn**
## Description
Lower the default API timeout from `30s` to `10s`, since it's rare that an API would successfully respond if it hasn't within the first 10 seconds. Open to decreasing this further, but we need to validate that it doesn't increase API timeout failures (the lower we go, the more likely timeouts will be triggered).

## Changes
- Decrease default `API_TIMEOUT` to `10000` (10s)
- Set the default override for Kaiko's `API_TIMEOUT` back to `30000` (30s)

## Quality Assurance
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
- [x] All code changes have 100% unit and integration test coverage. If testing is not applicable or too difficult to justify doing, the reasoning should be documented explicitly in the PR.

### CODE REVIEW

**Comment by justinkaseman** [comment]
Brendon has requested this in the past, so I'm sure he will be thrilled :grin:.

@austinborn We can test this on our monitoring before making the actual code change. Let's do that first so that we don't have to roll this back if any complications arise.

**Comment by nzleet** [comment]
Hi guys. I see no issue with this for all EAs other than Kaiko. We should really try and figure out what is causing Kaiko to require such a long timeout though - is it purely down to their API taking so long to respond?

**Comment by boxhock** [comment]
@austinborn To proceed with this PR, can we change the Kaiko default to still be 30s, while setting the framework default to 10s?

That will alleviate concerns around Kaiko for now, and gives us time to investigate this further.

**Comment by austinborn** [comment]
> @austinborn To proceed with this PR, can we change the Kaiko default to still be 30s, while setting the framework default to 10s?
> 
> That will alleviate concerns around Kaiko for now, and gives us time to investigate this further.

I've set Kaiko's default `API_TIMEOUT` back to `30000` through the `envDefaultOverrides`. Does it look alright?

File: .changeset/modern-apples-travel.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@chainlink/ea-bootstrap': patch
+'@chainlink/kaiko-adapter': patch
+---
+
+Add API_TIMEOUT to env default overrides, with override for Kaiko
```
Comment (boxhock) [inline .changeset/modern-apples-travel.md:6]: Feel free to re-phrase, just want to include the actual value in the changeset.

```suggestion
Add API_TIMEOUT to env default overrides, with an override of 30s for Kaiko
```

**Review by boxhock — CHANGES_REQUESTED** [review]
Just a tiny changeset request, otherwise LGTM!

File: .changeset/modern-apples-travel.md (hunk 1 above)
Comment (austinborn) [reply .changeset/modern-apples-travel.md]: Done!

**Review by boxhock — APPROVED** [review]

**Pull request merged**